go.sum

```diff
@@ -851,6 +851,8 @@ github.com/rancher/system-upgrade-controller v0.4.1-0.20200326220202-4655d4a551b
 github.com/rancher/system-upgrade-controller v0.4.1-0.20200326220202-4655d4a551bd/go.mod h1:8x2eK9Q5/1c2AC9xJtNFyAyjP9ytP4uPHhGA6wajIS0=
 github.com/rancher/types v0.0.0-20200723224053-ae84b42a4797 h1:luVtAWdaFjTc80CC117r2bwLj3xAwW9UgHXjoOpKKqU=
 github.com/rancher/types v0.0.0-20200723224053-ae84b42a4797/go.mod h1:HX+ryuo/4a4OtbjPzaCvXWL7nAk5sQN5dP9XQl6iH78=
+github.com/rancher/types v0.0.0-20200812201428-42b3abe8e39e h1:nW5niNDJD9wSHUMtzAor7DMqF1OFr/l7NZ6DINkcRTs=
+github.com/rancher/types v0.0.0-20200812201428-42b3abe8e39e/go.mod h1:HX+ryuo/4a4OtbjPzaCvXWL7nAk5sQN5dP9XQl6iH78=
```

Comment:
Do you need to run tidy ?

Reply:
hm I did run tidy, and confirmed again but nothing changed